virttest/utils_conn.py

```diff
@@ -1075,6 +1078,14 @@ def conn_recover(self):
 
         # restart libvirtd service on server
         try:
+            if self.server_setup_local:
+                libvirtd_service = utils_libvirtd.Libvirtd()
+                if libvirt_version.version_compare(5, 6, 0):
+                    process.run("systemctl stop libvirtd-tls.socket")
```

Comment:
Do we need to stop libvirtd.socket and libvirtd.service like before stop libvirtd-tls.socket? Just like what it do in this file...

Reply:
I think there is no need to stop libvirtd.socket and libvirtd.service before stop libvirtd-tls.socket. But I should use libvirtd_service.restart() instead of libvirtd_service.start() here.